scribus/plugins/import/pdf/slaoutput.cpp

```diff
@@ -3896,3 +3926,680 @@ bool SlaOutputDev::checkClip()
 	}
 	return ret;
 }
+
+// TODO: remove mutual dependencies and move the render engine into it's own cc and header files because it's getting hard to navigate the code base.
+// PDF never deviates from the line when it comes to colenear
+bool TextRegion::coLinera(qreal a, qreal b)

```

Comment:
i would say that this function needs a better name: what is `coLinera`?

colinear? `isColinear` is not really clear to me and you might need to use the doc before to say what it is or link to a definition.

and the body can simply be

```
{
   return abs(a - b) < 1;
}

Reply:
it's collinear my bad spelling. in geometry it means on the same line.

Reply:
In geometry, collinearity of a set of points is the property of their lying on a single line. A set of points with this property is said to be collinear. In greater generality, the term has been used for aligned objects, that is, things being "in a line" or "in a row".

Reply:
ok, i'll put soome documentation in where I've used uncommon words or common words uncommonly